Realm/RLMRealm.h

```diff
@@ -828,6 +828,20 @@ NS_REFINED_FOR_SWIFT;
  */
 - (void)addOrUpdateObjects:(id<NSFastEnumeration>)objects;
 
+/**
+ Creates an Asymmetric object, which will be sync'd unidirectionally and
+ cannot be query locally, only objects which inherits from `RLMAsymmetricObject`
+ can be created using this method.
+
+ Objects created using this method will not be added to the Realm.
+
+ @warning This method may only be called during a write transaction.
+
+ @param object The asymmetric object to create.
+
+ */
+- (void)createObject:(RLMAsymmetricObject *)object;
```

Comment:
This function is weird and different from the normal API for no obvious reason.

Reply:
I changed to instead of using the realm to create the object, you can do it like this `[PersonAsymmetric createInRealm:realm withObject:person]` like the others APIs